xbmc/music/Song.cpp

```diff
@@ -160,6 +160,27 @@ void CSong::Clear()
   bCompilation = false;
   embeddedArt.clear();
 }
+const std::vector<std::string> CSong::GetArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> songartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
+  {
+    songartists.push_back(artistCredit->GetArtist());
+  }
+  return songartists;
+}
+
+const std::vector<std::string> CSong::GetMusicBrainzArtistID() const
+{
+  //Get artist MBIDs as vector from artist credits
+  std::vector<std::string> MBID;
```

Comment:
Only constants should be all uppercase. http://kodi.wiki/view/Official:Code_guidelines_and_formatting_conventions#Constants
Not sure if this applies here too.